database/migrations/2024_10_19_172804_create_companies_table.php

```diff
@@ -11,10 +11,10 @@
     public function up(): void
     {
         Schema::create('companies', function (Blueprint $table) {
-            $table->id();
+            $table->uuid('id')->primary();
             $table->string('name', 255);
-            $table->string('email', 255);
-            $table->string('NIF', 255);
+            $table->string('email', 255)->unique();
```

Comment:
what happens if a person has an email and uses the same email in all the companies he has?